Sorting/SortingUtils.h

```diff
@@ -9,15 +9,24 @@
 
 #ifndef VECTOR_H
 #include <vector>
-#endif
+#include <cstdlib>
 
+#endif
+#define EXIT_CODE_ERROR -1
```

Comment:
Give it a more meaningful name such as `EXIT_SORT_NOT_REQUIRED`.

Also, since `-1` is the code used mostly for errors, give it a value of, say, `2`. (Since it is not _really_ an error if there are 0 elements to sort, right?)